README.md

```diff
@@ -10,33 +12,69 @@
 [![Known Vulnerabilities](https://snyk.io/test/npm/@ice/store/badge.svg)](https://snyk.io/test/npm/@ice/store)
 [![David deps](https://img.shields.io/david/ice-lab/icestore.svg?style=flat-square)](https://david-dm.org/ice-lab/icestore)
 
-## 安装
+## Installation
 
 ```bash
 npm install @ice/store --save
 ```
 
-## 快速上手
+## Introduction
+
+`icestore` is a lightweight React state management library based on hooks. It has the following core features:
+
+* Minimal API: Contains 3 APIs, which is easily learnable in 5 minutes.
+* Predictable: Uses unidirectional data flow (similar to Redux) and allows state mutation only inside actions, allowing data flow to be traced easily.
+* Optimal performance: decrease the number of view components needs to rerender when state changes by creating multiple stores, and rerendering only occurs when the state changes by diffing with previous state.
```

Comment:
Decreases the number of view components that rerender when the state changes by creating multiple stores. Rerendering only occurs when the current state is different from the previous state.